http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -692,6 +704,22 @@ public String getTokenSignatureAlgorithm() {
         return tokenSignatureAlgorithm;
     }
 
+    public boolean isSessionRequiredOnLogout() {
+        return sessionRequiredOnLogout;
+    }
+
+    public void setSessionRequiredOnLogout(boolean sessionRequiredOnLogout) {
+        this.sessionRequiredOnLogout = sessionRequiredOnLogout;
+    }
+
+    public int getLogoutSessionWaitingLimit() {
```

Comment:
Just thinking it might be good to rename this to make it easier to understand what it's used for.

Reply:
done